client/app/components/UserGuideDialog.vue

```diff
@@ -12,50 +12,53 @@ export default {
     return {
       categories: [
         {
-          name: 'Interactions',
+          name: 'General',
           data: [
             {
-              name: 'Select Detection', icon: 'mdi-mouse', actions: ['Left Click Mouse'], description: 'Left click a rectangle to select a detection/track',
+              name: 'Select Track', icon: 'mdi-mouse', actions: ['Left Click Mouse'], description: 'Left click a rectangle to select a detection/track',
             },
             {
               name: 'Zoom In/Out', icon: 'mdi-mouse', actions: ['Scrollwheel Up/Down'], description: 'use scrollwheel to zoom in and out',
             },
             {
               name: 'Zoom Area', icon: 'mdi-mouse', actions: ['Shift + Mouse Movement'], description: 'Zoom into a specific area',
             },
+            {
+              name: 'Reset zoom', icon: 'mdi-keyboard', actions: ['R Key'], description: 'Reset pan and zoom',
+            },
             {
               name: 'Select Track', icon: 'mdi-keyboard', actions: ['Up/Down Arrows'], description: 'Select Track',
```

Comment:
`name` is used as the v-for key and causes a Vue warn here about duplicate keys because there are now two data items with the `name='Select Track'`, just suggest changing to the v-for to  ```:key="`${item.name}_${index}`"```.